src/crimson/os/seastore/random_block_manager/rbm_device.cc

```diff
@@ -70,23 +70,32 @@ RBMDevice::mkfs_ret RBMDevice::do_primary_mkfs(device_config_t config, int shard
 }
 
 RBMDevice::mkfs_ret RBMDevice::do_shard_mkfs() {
```

Comment:
do_shard_mkfs duplicate with do_shard_mount only there is a close in shard_mkfs, so we can reuse the do_shard_mount code here.

Reply:
Done